.github/workflows/_test_build_windows.yaml

```diff
@@ -0,0 +1,77 @@
+name: Test Build Windows
+
+on:
+  workflow_run:
```

Comment:
Isn't this calling the job twice ?

Reply:
We switch the approach to have a workflow with the 2 dependent jobs.